tests/os.rs

```diff
@@ -124,7 +124,21 @@ fn syscalls_cairo1(block_context: BlockContext, initial_state: InitialState, max
         nonce: nonce_manager.next(sender_address),
     });
 
-    let txs = vec![test_emit_event_tx, test_storage_read_write_tx, test_get_block_hash_tx];
+    // test_send_message_to_l1
+
+    let to_address = stark_felt!(1234_u16);
+    let payload = vec![stark_felt!(2019_u16), stark_felt!(2020_u16), stark_felt!(2021_u16)];
+    let entrypoint_args = &[vec![to_address, stark_felt!(payload.len() as u64)], payload].concat();
+
+    let test_send_message_to_l1_tx = test_utils::account_invoke_tx(invoke_tx_args! {
+        max_fee,
+        sender_address: sender_address,
+        calldata: create_calldata(contract_address, "test_send_message_to_l1", entrypoint_args),
+        version: tx_version,
+        nonce: nonce_manager.next(sender_address),
+    });
+
+    let txs = vec![test_emit_event_tx, test_storage_read_write_tx, test_get_block_hash_tx, test_send_message_to_l1_tx];
```

Comment:
Unsure of how these tests work, but shouldn't there be at least an assertion somewhere to test that the code works?

Reply:
You mean that each individual transaction succeeded?

Reply:
Yes, and this does not not make sense in an integration test, thanks for the clarification.